src/model/eligibilityReasonCode.ts

```diff
@@ -0,0 +1,7 @@
+export enum eligibilityReasonCode {
+  COMPANY_IS_ELIGIBLE = "company is eligible",
```

Comment:
So these values don't seem to match the current return codes from the API - @mattch1 did merge a PR recently which changed these. So I think they need to correspond to what's in the API **_EligibilityReturnCodeToWeb_** Java enum.

Reply:
Yep, I'll change the values to match those in the api. The strings okay to stay as they are?

Reply:
🆗 Yes, string values are fine, assuming they're still sensible for the values in EligibilityReturnCodeToWeb.